pkg/activator/net/throttler.go

```diff
@@ -524,3 +526,23 @@ func (ib *InfiniteBreaker) Maybe(ctx context.Context, thunk func()) error {
 		return ctx.Err()
 	}
 }
+
+// SetCapacity is a test helper that sets Throttler in a desired
+// state for unit testing external components.
+// This permits us to hide behind the veneer the details of how Thorttler is updated.
+func (t *Throttler) SetCapacity(ns, n string, backends int, clusterIP string) {
+	upd := revisionDestsUpdate{
+		Rev:           types.NamespacedName{ns, n},
+		ClusterIPDest: clusterIP,
+		Dests:         dests(backends),
```

Comment:
This feels weird to me.  Maybe we can have the function name more clearly denote it's purpose for testing?

Reply:
Will do. In the parent change.